rampwf/tests/kits/iris/submissions/random_forest_10_10/classifier.py

```diff
@@ -1,18 +1,7 @@
-from sklearn.base import BaseEstimator
 from sklearn.ensemble import RandomForestClassifier
 
 
-class Classifier(BaseEstimator):
-    def __init__(self):
-        pass
-
-    def fit(self, X, y):
-        self.clf = RandomForestClassifier(
-            n_estimators=10, max_leaf_nodes=10, random_state=61)
-        self.clf.fit(X, y)
-
-    def predict(self, X):
-        return self.clf.predict(X)
-
-    def predict_proba(self, X):
-        return self.clf.predict_proba(X)
+def get_estimator():
```

Comment:
we are not consistent. Sometimes it's called submission.py and sometimes here it stayed as classifier.py

I would not call it submission as I find the work overloaded. Here we talk about the file that contains the estimator inside a submission. So maybe classifer.py everywhere?

Reply:
`pipeline.py` / `ml_pipeline.py`?

Reply:
It's not always a classifier so I'll change all submissions to `pipeline.py` - in which case I can make that the default in the `SKLearnPipeline` class.